user/libchewing/template.py

```diff
@@ -0,0 +1,47 @@
+pkgname = "libchewing"
+pkgver = "0.9.0"
+pkgrel = 0
+build_style = "cmake"
+# The test fails when run in parallel.
+make_check_env = {"CTEST_PARALLEL_LEVEL": "1"}
+pkgdesc = "Intelligent phonetic input method library"
+maintainer = "xunil-cloud <river_electron@proton.me>"
+license = "LGPL-2.1-only"
+url = "https://chewing.im"
+source = f"https://github.com/chewing/libchewing/releases/download/v{pkgver}/libchewing-{pkgver}.tar.zst"
+sha256 = "58e62cd0649ba3856ffa7c67560c1cfbcbb8713342a533f7700587b51efe84e3"
+tool_flags = {"CFLAGS": ["-DHAVE_NCURSES_H"]}
+
+hostmakedepends = [
+    "cmake",
+    "corrosion",
+    "ninja",
+    "cargo",
+    "pkgconf",
+]
+
+makedepends = ["sqlite-devel", "ncurses-devel"]
+
+depends = ["sqlite"]
```

Comment:
this isn't needed

Reply:
Ah, Thanks for the explanation. Will fix this later.